### PR TITLE
Smoketest the broker_examples module

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

**Comment by danielballan** [comment]
This is already covered, I believe: https://github.com/NSLS-II/bluesky/blob/master/bluesky/tests/test_vertical_integration.py#L48

**Comment by ericdill** [comment]
I'm not so sure that it is covered.  There is a syntax error in the broker_examples module! None of the det_* objects at the bottom of https://github.com/NSLS-II/bluesky/blob/master/bluesky/broker_examples.py get created because the imports of the motors were deleted!

**Comment by ericdill** [comment]
Err, maybe not "syntax" error, but the motor* objects don't exist in that module

**Comment by danielballan** [comment]
I was conflating broker_callbacks with broker_examples. Might as well add a meaningful tests here -- LiveMesh is one way to test SynGauss2D.

File: bluesky/examples.py [hunk 1]
```diff
@@ -21,6 +21,7 @@ def describe(self):
 
 class Base:
     def __init__(self, name, fields):
+        self._name = name
```
Comment (ericdill) [inline bluesky/examples.py]: @danielballan Something was expecting these objects to have a `_name` attribute.  I did not dig at all to see if the `.name` attribute is still used.  i guess the question here is should this be `_name` or `name`?

**Comment by ericdill** [comment]
Tests are failing on travis because of the tifffile import. That is being fixed in #341.  Other than that, I only have a question about `_name` vs `name`, which is that in-line comment.

File: bluesky/examples.py (hunk 1 above)
Comment (danielballan) [inline bluesky/examples.py]: It should be `name`, which is now part of the "bluesky interface" that all ophyd devices and signals are expected to have. Nothing should still be expecting a `_name` attribute.

Comment (danielballan) [inline bluesky/examples.py]: Looks like we failed to catch all the instances of `_name` in bluesky:

```
$ git grep -e "\._name"
bluesky/broker_examples.py:        self._data = {self._name: {'value': fs_uid, 'timestamp': ttime.time()}}
bluesky/broker_examples.py:        return {self._name: {'source': self._name,
doc/source/custom-plans.rst:                m=ret_m[motor._name]['value'], d=ret_d[det1._name]['value']))
```

Comment (ericdill) [inline bluesky/examples.py]: Ok, so I'll rename these `_name` to `name` in `broker_examples`

Comment (ericdill) [inline bluesky/examples.py]: And remove this change in `examples.py`. Thanks for the feedback

**Comment by ericdill** [comment]
restart travis after tifffile dep has been fixed

**Pull request merged**